hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -128,24 +128,23 @@ public Builder retainCommits(int commitsRetained) {
     public HoodieMetadataConfig build() {
       HoodieMetadataConfig config = new HoodieMetadataConfig(props);
       setDefaultOnCondition(props, !props.containsKey(METADATA_ENABLE_PROP), METADATA_ENABLE_PROP,
-          String.valueOf(DEFAULT_METADATA_ENABLE));
+              String.valueOf(DEFAULT_METADATA_ENABLE));
```

Comment:
trying to understand these formatting changes. can we use whichever aligns with checkstyle?

Reply:
Reverted it back. The earlier one aligns better with Hudi checkstyle.